### PR TITLE
Add Ocelot

### PR DESCRIPTION
**Pull request by Blackolf931**

### CODE REVIEW

File: Ocelot.API/Controllers/WeatherForecastController.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Ocelot.API.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
```
Comment (nantonov) [inline Ocelot.API/Controllers/WeatherForecastController.cs:12]: Get rid of WeatherForecastController

File: Ocelot.API/WeatherForecast.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Ocelot.API
+{
+    public class WeatherForecast
```
Comment (nantonov) [inline Ocelot.API/WeatherForecast.cs:5]: Get rid of it

File: Ocelot.API/Controllers/WeatherForecastController.cs (hunk 1 above)
Comment (Blackolf931) [reply Ocelot.API/Controllers/WeatherForecastController.cs]: deleted

File: Ocelot.API/WeatherForecast.cs (hunk 2 above)
Comment (Blackolf931) [reply Ocelot.API/WeatherForecast.cs]: deleted

**Review by nantonov — APPROVED** [review]

**Pull request merged**